### PR TITLE
Updated translations

### PR DESCRIPTION
**Pull request by null-none**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Bouke** [comment]
Translations are managed through Transifex.

File: user_sessions/locale/ru/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+"Вы также можете завершить все другие сеансы, кроме текущего.\n"
+"        Вы выйдете из системы на всех других устройствах.."
+
+#: templates/user_sessions/session_list.html:49
+msgid "Завершить все остальные сеансы"
+msgstr ""
+
+#: templatetags/user_sessions.py:20
+msgid "Chrome"
+msgstr ""
```
Comment (WhyNotHugo) [inline user_sessions/locale/ru/LC_MESSAGES/django.po:114]: Excuse my ignorance: does an empty string fall back to the original or render the actual empty string?

Comment (mbeijen) [reply user_sessions/locale/ru/LC_MESSAGES/django.po:114]: yes indeed, it falls back to the original tag!

**Review by WhyNotHugo — APPROVED** [review]
Thanks!

**Pull request merged**